website/docs/developer/deployment-guide/recovery-workflow.mdx

```diff
@@ -34,3 +34,100 @@ directly or by checking the status on the Cumulus Dashboard.
     zoomOutPic={useBaseUrl('img/zoom-out.svg')}
     resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
 />
+
+See the recovery workflow [stepfunction modules](https://github.com/nasa/cumulus-orca/tree/master/modules/workflows/OrcaRecoveryWorkflow) 
+for additional details on recovering files belonging to a granule.
+
+## Recovery workflow parameters
+
+The following are the parameters needed for recovery workflow:
+
+- buckets- AWS S3 bucket mapping used for Cumulus and ORCA configuration. Contains the following properties:
```

Comment:
Doesn't affect this card, but does anybody know why we're requiring this to be passed into the step function, and not just using the value from tfvars?

Reply:
maybe because it gives the user ability to override the values in terraform.

Reply:
This is likely some tech debt, in the original we pulled this information from the CMA and just kept it as part of the event. Long term it is probably better in the event as the bucket mapping is really something that is part of Cumulus and is used to help determine where to copy back to, we may  be able to tease this out or move it in future iterations and simplify this workflow.